DDDEastAnglia/Models/SponsorModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DDDEastAnglia.Models
+{
+    public class SponsorModel
```

Comment:
I think we now have three classes that represent this information. Are they all sufficiently distinct to warrant their separate existences?